server/src/endpoints/minidump.rs

```diff
@@ -0,0 +1,161 @@
+use actix_web::dev::Payload;
+use actix_web::{
+    actix::ResponseFuture, http::Method, multipart, pred, HttpMessage, HttpRequest, HttpResponse,
+};
+use futures::{
+    future::{self, Future},
+    Stream,
+};
+
+use future::ok;
+
+use crate::body::StorePayloadError;
+use crate::endpoints::common::{handle_store_like_request, BadStoreRequest};
+use crate::envelope::{ContentType, Envelope, Item, ItemType};
+use crate::extractors::{EventMeta, StartTime};
+use crate::service::{ServiceApp, ServiceState};
+use semaphore_general::protocol::EventId;
+use std::str::from_utf8;
```

Comment:
_nit_: Please order dependencies.

Reply:
This is ordered alphabetically. @RaduW We do this though:

```rust
// all std imports
use std::...;

// third party imports
use future::...;

// self-imports
use crate::...;
```

I would say `semaphore_general` counts as external dependency too.

Reply:
In semaphore, we have a dedicated group between third-party dependencies and crate-local imports.